sdks/open-format/src/core/contract.ts

```diff
@@ -27,6 +26,39 @@ export async function mint({
   return receipt;
 }
 
+export async function setRoyalties({
+  contractAddress,
+  signer,
+  royaltyReceiverAddress,
+  royaltyPercentage,
+}: ContractArgs & {
+  royaltyReceiverAddress: string;
+  royaltyPercentage: number;
+}) {
+  const openFormat = getContract({ contractAddress, signer });
+
+  const tx = await openFormat.setRoyalties(
+    royaltyReceiverAddress,
+    royaltyPercentage
+  );
+
+  const receipt = await tx.wait();
+
+  return receipt;
+}
+
+export async function getRoyalties({
+  contractAddress,
+  signer,
+  salePrice,
+}: ContractArgs & { salePrice: number }) {
+  const openFormat = getContract({ contractAddress, signer });
+
+  const tx = await openFormat.royaltyInfo(0, salePrice);
```

Comment:
I know this `0` was mentioned of always being.. zero. So thought this could be hardcoded rather than an arg.